bitnami/airflow/values.yaml

```diff
@@ -840,17 +840,27 @@ git:
 ## @param ldap.enabled Enable LDAP authentication
 ## @param ldap.uri Server URI, eg. ldap://ldap_server:389
 ## @param ldap.base Base of the search, eg. ou=example,o=org
+## @param ldap.uidField if doing an indirect bind to ldap, this is the field that matches the username when searching for the account to bind to
 ## @param ldap.binddn Bind DN
 ## @param ldap.bindpw Bind Password
-## @param ldap.uidField Field used for uid
+## @param ldap.userRegistration Set to True to enable user self registration
+## @param ldap.userRegistrationRole Set role name to be assign when a user registers himself. This role must already exist. Mandatory when using ldap.userRegistration
+## @param ldap.rolesMapping mapping from LDAP DN to a list of roles
+## @param ldap.rolesSyncAtLogin replace ALL the user's roles each login, or only on registration
+## @para
 ##
 ldap:
   enabled: false
-  uri: 'ldap://ldap_server:389'
-  base: 'ou=example,o=org'
-  binddn: 'cn=user,ou=example,o=org'
-  bindpw: ""
-  uidField: 'uid'
+  uri: 'ldap://ldap_server:1389'
+  base: 'dc=example,dc=org'
+  uidField: 'cn'
+  binddn: 'cn=admin,dc=example,dc=org'
+  bindpw: ''
```

Comment:
Is there any reason why those default parameters were modified? For example, why is the port different now?

If there is a user that installed the existing version of the chart with `ldap.enabled=true`, is a simple `helm upgrade` going to work with those changes or his LDAP configuration is going to be broken because it is different right now?

Reply:
The previous existing parameters do not work properly with OpenLDAP, let me explain it:
* They were not enough in order to get an Airflow+OpenLDAP integration working. Whatever values you used here it produced the ERR_TOO_MANY_REDIRECTS error.
* _uri port_: we can modify it to 389 because this is the OpenLDAP default. No problem with that. should I proceed?
* _base_: since every user is going to use a different OpenLDAP scheme we should provide default values aligned with a basic OpenLDAP example, typically 'example.org'. According to the X.500 Directory Specification 'example.org' dn should be 'dc=example,dc=org' since they are 'domain components', an OpenLDAP search is going to look for the object inside the 'org' domain component, and then inside 'example' domain component. In this case it suggests that there is a 'admin' object inside the 'example' domain component. Each part of a DNS domain is called DC in OpenLDAP and they should be used as that in an object DN (distinguished name) like 'cn=admin,dc=example,dc=org'. In fact what is wrong is the current existing value since it is not fit neither the OpenLDAP standard nor the typical 'dn' value expected by an OpenLDAP administrator.
* Regarding upgrades: current Airflow+OpenLDAP integrations are not working since the current Chart is not configuring a proper webserver_config.py file. This is because the current values are not enough like explained in the first point.

Reply:
Understood, it makes sense; thanks. My 2cts: maintain those values that don't need to be modified to make it works like the port, the rest of parameters whatever value that works is fine

Reply:
ok port fixed! thank you @carrodher !